src/utils.rs

```diff
@@ -40,6 +42,27 @@ pub(crate) fn extract_app_lookup(extension: &HeaderExtension) -> &DataLookup {
 	}
 }
 
+pub fn filter_auth_set_changes(header: DaHeader) -> Vec<Vec<(AuthorityId, u64)>> {
```

Comment:
Do we need a test for this func?

Reply:
Well, maybe in avail-subxt, or similar. Creating a fake header here with random digest for testing seems odd.